code/game/objects/items/devices/traitordevices.dm

```diff
@@ -355,11 +355,13 @@ effective or pretty fucking useless.
 	user.balloon_alert(user, "disruptor wave released!")
 	to_chat(user, span_notice("You release a disruptor wave, disabling all nearby radio devices."))
 	for (var/atom/potential_owner in view(7, user))
-		disable_radios_on(potential_owner)
+		disable_radios_on(potential_owner, ignore_syndie = TRUE)
 	COOLDOWN_START(src, jam_cooldown, jam_cooldown_duration)
 
 /obj/item/jammer/attack_self_secondary(mob/user, modifiers)
 	. = ..()
+	if(. == SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN)
```

Comment:
```suggestion
	if(.)
```

Reply:
Pretty sure they had it right the first time

Reply:
(But it doesn't *really* matter I suppose)

Reply:
> Pretty sure they had it right the first time

technically yes but everything uses if(.) so this is just for code consistency's sake